azurerm/helpers/authentication/validation.go

```diff
@@ -2,11 +2,54 @@ package authentication
 
 import (
 	"fmt"
+	"log"
 
+	"github.com/Azure/go-autorest/autorest/adal"
 	"github.com/hashicorp/go-multierror"
 )
 
-func (c *Config) ValidateBearerAuth() error {
+func (c *Config) Validate() error {
```

Comment:
Nit: Exported functions should be documented

Reply:
done 👍